components/display/touch_panel/ft5x06/ft5x06.c

```diff
@@ -240,6 +240,20 @@ esp_err_t ft5x06_set_direction(touch_panel_dir_t dir)
 
 int ft5x06_is_press(void)
 {
+    /**
+     * @note There are two ways to determine weather the touch panel is pressed
+     * 1. Read the IRQ line of touch controller
+     * 2. Read number of points touched in the register
+     */
+    if (-1 != g_dev.pin_num_int) {
+        /**
+         * @note Detect the no touch case using the IRQ line
+         * If a touch is detected, fall through to read the register to confirm no multi touch
+         */
+        if (gpio_get_level((gpio_num_t)g_dev.pin_num_int)) {
+            return 0;
+            }
+        }
```

Comment:
```suggestion
    if (-1 != g_dev.pin_num_int) {
        /**
         * @note Detect the no touch case using the IRQ line
         * If a touch is detected, fall through to read the register to confirm no multi touch
         */
        return !gpio_get_level((gpio_num_t)g_dev.pin_num_int);
    }
```
It can be consistent with other modifications.

Reply:
Thanks for the feedback @Lzw655. I was trying to preserve the current behaviour of the driver, which is to return 'no touch' if there are multiple points pressed. I see that the driver does support multiple touches, so my caution is likely misplaced. I have incorporated your suggestion.